### PR TITLE
Edit football-data API

### PR DESCRIPTION
**Pull request by william-nod**
- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -933,7 +933,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Fitbit](https://dev.fitbit.com/) | Fitbit Information | `OAuth` | Yes | Unknown |
 | [Football (Soccer) Videos](https://www.scorebat.com/video-api/) | Embed codes for goals and highlights from Premier League, Bundesliga, Serie A and many more | No | Yes | Yes |
 | [Football Prediction](https://boggio-analytics.com/fp-api/) | Predictions for upcoming football matches, odds, results and stats | `X-Mashape-Key` | Yes | Unknown |
-| [Football-Data.org](http://api.football-data.org/index) | Football Data | No | No | Unknown |
+| [Football-Data.org](http://api.football-data.org/index) | Football Data with match info, player, team, and competitions | `X-Mashape-Key` | Yes | Unknown |
```
Comment (pawelborkar) [inline README.md:936]: /

Comment (pawelborkar) [reply README.md:936]: No need to add the TLD ref. [contribution guide](https://github.com/public-apis/public-apis/blob/master/CONTRIBUTING.md)

**Review by pawelborkar — CHANGES_REQUESTED** [review]
Thanks for opening this PR @WiLLiaM-noD 
Please commit the suggestions stated below and don't forget to squash all the commits into a single one

File: README.md (hunk 1 above)
Comment (pawelborkar) [inline README.md:936]:
```suggestion
| [Football-Data](http://api.football-data.org/index) | Football Data with match info, player, team, and competitions | `X-Mashape-Key` | Yes | Unknown |
```

Comment (pawelborkar) [reply README.md:936]: No need to add the TLD ref [contribution.md](https://github.com/public-apis/public-apis/blob/master/CONTRIBUTING.md)

**Review by pawelborkar — CHANGES_REQUESTED** [review]

**Comment by WiLLiaM-noD** [comment]
I've already changed the API name.

**Comment by pawelborkar** [comment]
Please squash all the commits into a single one. @WiLLiaM-noD

**Comment by WiLLiaM-noD** [comment]
Finished squash all the commit

**Comment by pawelborkar** [comment]
The commits are not squashed as you can see in the commit tab of this PR.
I would encourage you to make a new PR with the suggested changes and don't forget to mention this PR in the new one.

Thanks.

**Comment by pawelborkar** [comment]
Hi @william-nod ,
Please update the PR with the suggestions stated above.

**Pull request closed without merging**